build/azure-pipelines/linux/product-build-linux.yml

```diff
@@ -312,6 +312,9 @@ steps:
       continueOnError: true
       displayName: Download ESRPClient
 
+    - script: node build/azure-pipelines/common/sign $(Agent.ToolsDirectory)/esrpclient/*/*/net6.0/esrpcli.dll rpm $(ESRP-PKI) $(esrp-aad-username) $(esrp-aad-password) .build/linux/debian '*.deb'
```

Comment:
I am not sure if the Keycode in https://github.com/microsoft/vscode/blob/ba35c622b06307173c04184e0e270673d750a4d2/build/azure-pipelines/common/sign.ts#L39 applies to debian as well. Let me pull up the ESRP docs to confirm.

Reply:
Yeah looks good per https://microsoft.sharepoint.com/teams/prss/esrp/info/SitePages/Linux%20GPG%20Signing.aspx